### PR TITLE
LPS-118061 Allow sorting tags by number of usages or created date

### PR DESCRIPTION
**Pull request by javierdearcos**

### CODE REVIEW

File: modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/utils/client.es.js [hunk 1]
```diff
@@ -146,7 +146,47 @@ export const deleteMessageBoardThreadQuery = gql`
...
+			query: getTagsOrderByNumberOfUsagesQuery,
+			variables: {
+				page,
+				pageSize,
+				search,
+				siteKey,
+			},
+		})
+		.then((result) => ({
+			...result,
```
Comment (nhpatt) [inline modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/utils/client.es.js:184]: I don't think this is needed (I know we are doing it in other query calls) but I think you just need the data and you can unwrap it with ({data: {keywords}}) or something like that.

But it's ok, it's a good practice to return everything.

File: modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/pages/tags/Tags.es.js [hunk 2]
```diff
@@ -12,21 +12,38 @@
...
+import {
+	dateToInternationalHuman,
+	historyPushWithSlug,
+	useDebounceCallback,
+} from '../../utils/utils.es';
+
+function getOrderByOptions() {
+	return [
+		{
+			label: Liferay.Language.get('latest-created'),
```
Comment (nhpatt) [inline modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/pages/tags/Tags.es.js:38]: We should do a util for this.

File: modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/pages/tags/Tags.es.js [hunk 3]
```diff
@@ -64,24 +96,60 @@ export default withRouter(
...
+										id="tagsOrderBy"
+										onChange={(event) => {
+											setLoading(true);
+											setOrderBy(event.target.value);
+										}}
+										value={orderBy}
+									>
+										{orderByOptions.map((option) => (
+											<ClaySelect.Option
+												key={option.value}
```
Comment (nhpatt) [inline modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/pages/tags/Tags.es.js:133]: here I think you can spread option, do something like {...option} except for the key, damn.

**Comment by nhpatt** [comment]
ci:forward

**Comment by javierdearcos** [comment]
> Added some comments but just FYI, pull looks perfect :) Very good 👏 👏 👏

Thanks!!! Not very sure about what I was doing. Just trying to follow similar patterns to achieve the desired functionality

**Comment by nhpatt** [comment]
Uy, react tests failed :(

**Comment by javierdearcos** [comment]
> Uy, react tests failed :(

I will check it in a moment, when I finished a test for other issue

**Comment by javierdearcos** [comment]
ci:forward

**Pull request closed without merging**